app/assets/stylesheets/flash.scss

```diff
@@ -0,0 +1,27 @@
+.flash {
+  padding: 8px 35px 8px 14px;
```

Comment:
Properties should be ordered background-color, border, border-radius, color, margin-bottom, padding, text-shadow